### ISSUE TITLE
[Bug]: ‘show backend servers’ error

### ISSUE THREAD

**Issue by LiSong0214** [matrixorigin/matrixone#9473]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93):
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

![image](https://github.com/matrixorigin/matrixone/assets/31049447/e26623a7-1a34-4f4f-a729-a7926937963e)

**Comment by volgariver6** [comment]
cannot repro it, please provide steps how to repro.

**Comment by LiSong0214** [comment]
mo cloud platform - sql editor

**Comment by volgariver6** [comment]
fixed by #9999
